helpers/uuid.go

```diff
@@ -0,0 +1,49 @@
+package helpers
+
+import (
+	"fmt"
+
+	"github.com/satori/go.uuid"
+)
+
+// GenerateUUIDv4AsString returns random generated canonical string representation of UUID:
+// xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx.
+func GenerateUUIDv4AsString() string {
```

Comment:
why not NewUUID?

where `GenerateUUIDv1AsString`, `GenerateUUIDv2AsString`, `GenerateUUIDv3AsString`...?

Reply:
in more detail

Reply:
explain?